packages/compiler-cli/src/ngtsc/annotations/src/directive.ts

```diff
@@ -109,15 +131,14 @@ export class DirectiveDecoratorHandler implements
  * the module.
  */
 export function extractDirectiveMetadata(
-    clazz: ClassDeclaration, decorator: Decorator, reflector: ReflectionHost,
+    clazz: ClassDeclaration, decorator: Decorator | null, reflector: ReflectionHost,
```

Comment:
```suggestion
    clazz: ClassDeclaration, decorator: Decorator|null, reflector: ReflectionHost,
```

Surprised clang-format doesn't do this.

Reply:
Actually it seems to reformat it to the one with spaces when I remove the space. I'll leave it as is.